src/sources/exec/mod.rs

```diff
@@ -501,7 +501,7 @@ fn spawn_reader_thread<R: 'static + AsyncRead + Unpin + std::marker::Send>(
     sender: Sender<((SmallVec<[Event; 1]>, usize), &'static str)>,
 ) {
     // Start the green background thread for collecting
-    Box::pin(tokio::spawn(async move {
+    let _ = Box::pin(tokio::spawn(async move {
```

Comment:
Is there something better to do here? It was complaining that we weren't using the result of `Box::pin`.